odxtools/envdatadesc.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2022 MBition GmbH
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+from .globals import logger
+
+
+@dataclass
+class EnvDataDesc:
+    """This class represents a ENV-DATA-DESC."""
```

Comment:
same as the previous comment: I'd highly appreciate a bit more verbosity here...

Reply:
Updated with more descriptive text